Documentation/instruction_name_tables.md

```diff
@@ -0,0 +1,62 @@
+## Database table naming rules
+
+Before uploading a new dataset in the database and creating a new table, the following rules should be followed to name the table:
+
+### All names:
+1. Always in lower case; 
+2. No space between the characters. Instead of using space, use `_` to separate composite names.
+3. Column names should follow the above rules.
+
+
+### Schema names:
+
+If the table it's single country data use the full ISO country name. This name for each country of the world can be found [here](https://gadm.org/maps.html).
+
+If the data is on a global scale data use the schema: `global`.
+
+
+### Table names:
+
+Table names should follow this basic structure:
+
+`<locality>_<semantic single word name>_<0|D|W|M>_<optional suffix>`
+
+The symbols <> are just delimiters. On PostGIS tables representing maps, a `gis` prefix should be added. 
```

Comment:
```suggestion
The symbols `<>` are just delimiters for the placeholder, it means that one of the values inside should be used.

On PostGIS tables, when representing maps, a `gis` prefix should be added. 
```


`On PostGIS tables, when representing maps, a `gis` prefix should be added. ` it would be good to have an example for this case.

Reply:
Agreed on the need of an example.